README.md

```diff
@@ -0,0 +1,10 @@
+### Bash script to change dates and rename files
+
+remove_duplicate_files_with_paren1paren.sh
+
+This is a command-line Bash script to remove all files that contain:  (1).jpg
+e.g., IMG_0375 (1).jpg. 
```

Comment:
Another similarly small thing, but you can make those file name fragments appear in monospaced font inline by surrounding them with single backticks. It looks like `(1).jpg` and `IMG_0375 (1).jpg`.

Reply:
@kcharter Okay, I didn't know file name fragments should be in monospace fonts. Good to know conventions & how to do in markdown.